### DISCUSSION TITLE
Custom report: aggregate timespent from linked issues

### DISCUSSION THREAD

**Discussion by GYolshin** [shridhar-tl/jira-assistant#310] [General]
I'm happy to use custom report in JA, I like that I have possibility to insert javascript to calculate column values. Recently I faced with task to display timespent from linked issues. Is there any way to implement that in Custom reports?

**Comment by shridhar-tl** [comment]
Hi @GYolshin, using custom scripts, you can use and transform the value of data received from Jira. Linked issues would be available in the data from Jira and so you can use it in custom scripts. However, Jira does not send the worklog related information for linked issues and so you will not be able to get that information using script due to the fact that the information is not available in the context.

There is no way you can achieve this using custom report. But, this is achievable using Report builder. You can create a separate dataset based on a http request and achieve it. But due to lack of documentation it might be bit difficult understanding how to use it.

Some minimal documentation can be found in this url: https://www.jiraassistant.com/features/report-builder
